src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -488,6 +491,76 @@ void generateOverallSummaryFiles(
     generateSummaryJson(roundTallies, tallyTransfers, null, outputPath);
   }
 
+  // write CastVoteRecords for all contests to the same folder as the csvInputFile
+  // create the folder if needed
+  // folder will default to the input Dominion csv folder
```

Comment:
This function isn't specific to Dominion is it? If so, recommend removing this comment, particularly since it's already mentioned above that it writes to the same folder as the csvInputfile.